cpp/ps0005/main.cpp

```diff
@@ -3,41 +3,44 @@ using namespace std;
 
 int main()
 {
-    float A;
-    cout << "Please enter A ";
-    cin >> A;
+    // prompting user for the 2 operands and an operator 2/2
+    float num1;
+    cout << "Please enter number 1: ";
+    cin >> num1;
 
-    float B;
-    cout << "Please enter B ";
-    cin >> B;
+    float num2;
+    cout << "Please enter number 2: ";
+    cin >> num2;
 
     char op;
     cout << "Please enter an operator ";
     cin >> op;

```

Comment:
- [x] declared variables to store the user input
- [x] chose suitable data types for the variables
- [x] prompted user to enter data
- [x] used `std::cin` in a right way to get user's input